### PR TITLE
[IMP] account_bank_statement_import_paypal: create paypal maps

### PR DESCRIPTION
**Pull request by cubells**
Several changes to allow create Paypal templates. Now user can create a map for any Paypal statement he downloads:
- new model to set template and import headers easily.
- updated README.
- added tests.
- added translation.
- fix several bugs.

cc @Tecnativa

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
Too many changes. Some people would say you to separate the different changes into several commits or several PRs 😏 😂 But I won't say you that. Good work.

**Comment by pedrobaeza** [comment]
Hehe, take into account that this change in monolithic, as everything is needed for this to work. Are you reviewing it? :pray:

File: account_bank_statement_import_paypal/models/account_bank_statement_import_paypal_map.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    _name = 'account.bank.statement.import.paypal.map'
+
+    name = fields.Char(
+        required=True,
+    )
+    map_line_ids = fields.One2many(
+        comodel_name='account.bank.statement.import.paypal.map.line',
+        inverse_name='map_parent_id',
+        string="Map lines",
+        required=True,
```
Comment (JordiBForgeFlow) [inline account_bank_statement_import_paypal/models/account_bank_statement_import_paypal_map.py:17]: add 'copy=True'

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Technical and functional review. Tested with a real paypal statement. Very good addition!

File: account_bank_statement_import_paypal/models/account_bank_statement_import_paypal_map.py (hunk 1 above)
Comment (cubells) [reply account_bank_statement_import_paypal/models/account_bank_statement_import_paypal_map.py:17]: @jbeficent done.

File: account_bank_statement_import_paypal/data/paypal_map_data.xml [hunk 2]
```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" ?>
```
Comment (MiquelRForgeFlow) [inline account_bank_statement_import_paypal/data/paypal_map_data.xml]: `<?xml version="1.0" encoding="utf-8"?>`

File: account_bank_statement_import_paypal/views/account_journal_views.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" ?>
```
Comment (MiquelRForgeFlow) [inline account_bank_statement_import_paypal/views/account_journal_views.xml]: same as before

File: account_bank_statement_import_paypal/views/paypal_map_views.xml [hunk 4]
```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" ?>
```
Comment (MiquelRForgeFlow) [inline account_bank_statement_import_paypal/views/paypal_map_views.xml]: same as before

File: account_bank_statement_import_paypal/wizards/account_bank_statement_import_view.xml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" ?>
```
Comment (MiquelRForgeFlow) [inline account_bank_statement_import_paypal/wizards/account_bank_statement_import_view.xml]: same as before

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM code review

**Comment by MiquelRForgeFlow** [comment]
Will you migrate this module to v12.0?

**Comment by pedrobaeza** [comment]
If you want, all yours :wink:

**Comment by MiquelRForgeFlow** [comment]
No, I was just asking 😅

**Comment by pedrobaeza** [comment]
Well, this one is part of OCA, so someone has to do it. Any way, it should be very straight-forward

**Comment by pedrobaeza** [comment]
Migrated in #207

**Pull request merged**